hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClientWriteHandler.java

```diff
@@ -60,17 +61,13 @@ public void handle() {
             ready = true;
             return;
         }
-        try {
-            writeBuffer();
-        } catch (Throwable t) {
-            logger.severe("Fatal Error at WriteHandler for endPoint: " + connection.getEndPoint(), t);
-        } finally {
```

Comment:
The behaviorial change is intentional and that is why I need your input. On the serverside, any exception thrown in the read/write-handler automatically leads to a closing of the connection because you can't guarantee anything of the health of the connection.